content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -161,9 +185,9 @@ column.
 Rental rules tab
 ----------------
 
-Under the :guilabel:`Rental rules` tab, specific price rules can be configured for various rental
-products, using the same methodology as the :guilabel:`Price Rules` and :guilabel:`Recurring Prices`
-tabs.
+Price rules can be configured for :doc:`rental products <../../../sales/rental>` under the
```

Comment:
[CI error](https://runbot.odoo.com/runbot/build/74192048)
when we hop back three directories, we're still in the sales section so no need for it here.
```suggestion
Price rules can be configured for :doc:`rental products <../../../rental>` under the
```